playground/infrastructure/cd_helper.py

```diff
@@ -28,27 +37,106 @@ def store_examples(self, examples: List[Example]):
         """
         Store beam examples and their output in the Google Cloud.
         """
-        self._run_code(examples)
-        self._save_to_cloud(examples)
+        asyncio.run(self._get_outputs(examples))
+        self._save_to_cloud_storage(examples)
+        self._clear_temp_folder()
 
-    def _run_code(self, examples: List[Example]):
+    async def _get_outputs(self, examples: List[Example]):
         """
-        Run beam examples and keep their ouput.
+        Run beam examples and keep their output.
 
         Call the backend to start code processing for the examples. Then receive code output.
 
         Args:
             examples: beam examples that should be run
         """
-        # TODO [BEAM-13258] Implement logic
-        pass
+        await get_statuses(examples)  # run examples code and wait until all are executed
+        client = GRPCClient()
+        tasks = [client.get_run_output(example.pipeline_id) for example in examples]
+        outputs = await asyncio.gather(*tasks)
+        for output, example in zip(outputs, examples):
+            example.output = output
+
+    def _save_to_cloud_storage(self, examples: List[Example]):
+        """
+        Save examples, outputs and meta to bucket
+
+        Args:
+            examples: precompiled examples
+        """
+        self._storage_client = storage.Client()
+        self._bucket = self._storage_client.bucket(Config.BUCKET_NAME)
```

Comment:
Should we create a client each time for each `_save_to_cloud_storage` call or it doesn't matter?

Reply:
Hah, I recently changed it because somebody told that no need to create client every time)
I think it is better this way

Reply:
Got it, thanks!